### PR TITLE
edit seat

### PR DESCRIPTION
**Pull request by namframgia**

### CODE REVIEW

File: app/Http/Controllers/Admin/DiagramController.php [hunk 1]
```diff
@@ -144,20 +148,21 @@ public function saveLocation(Request $request, $id)
 
     public function saveAjaxLocation(Request $request)
     {
-        $data = $this->locationRepository->create([
-            'name' => $request->name,
-            'workspace_id' => $request->id,
-            'color' => $request->color,
+        $this->validate($request, [
+            'id' => 'required',
+            'seat' => 'required'
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/DiagramController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 153</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/Admin/DiagramController.php [hunk 2]
```diff
@@ -286,6 +291,11 @@ public function editInfoUser(Request $request)
 
     public function editSeat(Request $request)
     {
+        $this->validate($request, [
+            'seat_id' => 'required',
+            'color' => 'required'
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/DiagramController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 296</br>Each line in an array declaration must end in a comma

**Comment by namframgia** [comment]
https://edu-redmine.framgia.vn/issues/4150

File: public/js/permission-role.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                            '<input type="hidden" id="checked-' +
+                            data.permissionId +
+                            '" value="1">';
+
+                        $string =
+                            $string +
+                            '<i id="action-' +
+                            data.permissionId +
+                            '"\
+                                                class="fa fa-check-circle update-permission"\
```
Comment (BunnyPi04) [inline public/js/permission-role.js]: sao chỗ này tab rộng thế?

**Pull request merged**